simulator-core/src/test/java/com/evbox/everon/ocpp/mock/expect/RequestExpectationManager.java

```diff
@@ -20,8 +19,7 @@
      * @return optional of expected response.
      */
     public Optional<Function<Call, String>> findExpectedResponse(Call incomingRequest) {
-
-        List<RequestMatcher> matchers = getSuccessfulMatchers(incomingRequest);
-        return matchers.isEmpty() ? Optional.empty() : Optional.of(matchers.get(0).getExpectedResponse());
+        Optional<RequestMatcher> matcher = getSuccessfulMatcher(incomingRequest);
```

Comment:
Can be simplified to:
`return getSuccessfulMatcher(incomingRequest).map(matcher -> Optional.of(matcher.getExpectedResponse())).orElse(Optional.empty())`